RealifeTech-CoreSDK/GraphQL/API/Types.graphql.swift

```diff
@@ -114,6 +123,9 @@ public enum ApolloType {
         case (.nr, .nr): return true
         case (.de, .de): return true
         case (.sv, .sv): return true
+        case (.nb, .nb): return true
```

Comment:
Should be able to write this as comma separated. I have left a similar comment on another piece of code.